### PR TITLE
Add example of multi-version JSON file

### PR DESCRIPTION
**Pull request by drmacro**

### CODE REVIEW

**Comment by infotexture** [comment]
@drmacro Thanks for submitting this. 👍 

Please [sign your work](http://www.dita-ot.org/DCO) to give us permission to use  your contribution.

To sign off your last commit, use:

```
git commit --amend --signoff
```

Then force-push to update the branch on your fork with:

```
git push --force-with-lease
```

File: topics/plugins-registry.dita [hunk 1]
```diff
@@ -180,6 +180,20 @@
...
+   ...
+   "vers": "2.1.0",
+   ...
+ }<b>,</b>
+ { "name": "org.example.myplugin",
+   ...
+   "vers": "1.0.1",
+   ...
+ }
+]</codeblock></p>
```
Comment (infotexture) [inline topics/plugins-registry.dita]: @drmacro This doesn't look valid (or intentional) — perhaps a cursor placement issue in the authoring environment?

Title must be the first element in a section, but the new content is inserted here between the opening `<section>` element and the title.

I assume the intention was to place it at the end of the preceding section?

Comment (drmacro) [reply topics/plugins-registry.dita]: Yes, I didn't intend to split the section. Must have been an issue with the web author not making the new context clear. Or I was just in a hurry.

Comment (drmacro) [reply topics/plugins-registry.dita]: I meant to ask George to add a "Sign commit" to the automatic pull request creation that web author does but I didn't get a chance at the conference.

**Comment by infotexture** [comment]
OK, I'll merge as-is and take it from here.

**Pull request merged**